misc/config_tools/configurator/packages/configurator/src/pages/Config.vue

```diff
@@ -211,7 +211,14 @@ export default {
         }
       }
       if (this.currentFormData['load_order'] === 'SERVICE_VM') {
-        delete this.currentFormSchema.BasicConfigType.properties.vm_type
+        if (this.currentFormData.hasOwnProperty('vm_type')) {
+          delete this.schemas.ServiceVM.BasicConfigType.properties.vm_type
+        } else {
+          if (this.schemas.ServiceVM.BasicConfigType.properties.hasOwnProperty('vm_type') === false) {
+            this.schemas.ServiceVM.BasicConfigType.properties.vm_type =
+              {$ref: '#/definitions/BasicVMType', title: 'VM type', description: '<p>Select the VM type. A standard VM (<span class=…ial features for time-sensitive applications.</p>'}
```

Comment:
These tooltip/descriptions should be pulled from the .xsd files and not maintained separately in these .vue files, right?  @junjiemao1 @NanlinXie

Reply:
I'm not sure what this is for, but descriptions should always come from XSDs rather than being hardcoded.